src/ButtonLink/index.js

```diff
@@ -0,0 +1,210 @@
+// @flow
+import * as React from "react";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+import { withTheme } from "theming";
+
+import { iconSizes } from "../Icon";
+
+type Props = {
+  children?: React.Node,
```

Comment:
if the children is optional, I would add such example into stories

Reply:
OK, good point